vobject/vcard.py

```diff
@@ -147,7 +149,7 @@ def encode(cls, line):
         if not line.encoded:
             encoding = getattr(line, 'encoding_param', None)
             if encoding and encoding.upper() == cls.base64string:
-                line.value = line.value.encode('base64').replace('\n', '')
+                line.value = codecs.encode(line.value.encode(coding), "base64").decode("utf-8")
```

Comment:
The variable 'coding' is not defined? Is this a typo?
It looks like when my vcard has a photo which is base64 encoded, this breaks vcard.serialize().

I'm attaching a sample vcard test_file. The vcard has photo data encoded in base64 format.
A simple vcard.readOne(...).serialize() will fail.
[sample_vcard.txt](https://github.com/eventable/vobject/files/591494/sample_vcard.txt)